### PR TITLE
improve german translation by fixing typos phrases

### PR DESCRIPTION
**Pull request by wrosenauer**

### CODE REVIEW

File: assets/l10n/de_DE.po [hunk 1]
```diff
@@ -153,22 +153,22 @@ msgstr "Abbrechen"
...
+msgstr "Verschlüsselungseinstellungen für diesen Kontakt geändert"
 
 msgid "Chat"
 msgid_plural "Chats"
 msgstr[0] "Chat"
 msgstr[1] "Chats"
 
 msgid "Choose a provider from the list below to create a new account"
-msgstr "Wähle einen Anbieter aus der Liste um einen neues Konto zu erstellen"
+msgstr "Wähle einen Anbieter aus der Liste, um einen neues Konto zu erstellen"
```
Comment (florianhaar) [inline assets/l10n/de_DE.po]:
```suggestion
msgstr "Wähle einen Anbieter aus der Liste, um ein neues Konto zu erstellen"
```

Comment (wrosenauer) [reply assets/l10n/de_DE.po]: right, missed that ;-)

**Comment by Boehrsi** [comment]
Thanks for the fixed translations. We just received an updated version of all our translations which I will integrate right now. I will rebase the changes onto those adjustments and will integrate the diff.

**Comment by Boehrsi** [comment]
I just rebased everything on top of our latest translation changes. Please have a look @robert-virkus https://github.com/open-xchange/ox-coi/commit/88acc2cfa7d34d06ce9b3a3f64c7a1b822a4624e 
Please mark all translations which shouldn't be applied, so I will remove them from the commit and will integrate the other ones. Thanks again @wrosenauer for submitting this PR and finding all those typos.

**Comment by robert-virkus** [comment]
For one msgid and msgstr the parameters have now been left out, otherwise fine

**Comment by Boehrsi** [comment]
Merged by https://github.com/open-xchange/ox-coi/commit/7e76ca508a17ced27a816897a0d669e0ee891efb

**Pull request closed without merging**